apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -18,26 +39,32 @@ const props = defineProps<ComponentCardProps>();
     :class="{ 'card--clickable': props.href }"
     :href="props.href"
   >
-    <Badge
-      v-if="props.implemented"
-      class="card__badge card__badge--implemented"
-      text="Implemented"
-    />
-    <Badge v-else class="card__badge" text="Planned" type="info" />
-    <h4 class="card__title">{{ props.name }}</h4>
+    <ComponentStatusBadge :status="props.status" class="card__status" />
+    <OnyxHeadline is="h2">{{ props.name }}</OnyxHeadline>
+    <p class="estimation">
+      Estimation:
+      <span class="estimation__value">{{ estimationValue }}</span>
+    </p>
   </component>
 </template>
 
 <style lang="scss" scoped>
```

Comment:
Oh no.. In the figma link of the ticket, the lower variant was planned to be implemented. It is the one with this layout:
<img width="696" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/8620e670-585b-49a7-a92d-dccf20ff90d8">
You implemented the other variant of the tile design..

That's the reason why the refined ticket included a screenshot but somehow the screenshot is gone from the ticket now....

Reply:
Done :) Please review again